src/docs/product/relay/options.mdx

```diff
@@ -199,6 +199,42 @@ See also `cache.envelope_expiry`.
 
 Interval for evicting outdated project configs from memory.
 
+## Spooling
+
+These settings control on-disk spooling. Currently only spooling for incoming envelopes is supported.
+
+`spool.envelopes.path`
+
+: _String, default: null_
+
+  The path to the spool DB file, which is used to persist the envelopes to disk.
+  If provided enables on disk spooling.
+
+`spool.envelopes.max_memory_size`
+
+: _String, default: `500MB`_
```

Comment:
Units are separated from the value (see examples above), so either
```suggestion
: _String, default: `500` (MB)_
```
or
```suggestion
: _String, default: `500` (500MB)_
```
Same with the unit below.

Reply:
Provided values are correct, see for example how the `limits` below configured. 
This is the format (human readable) which is used to configure this options.